src/category_theory/abelian/exact.lean

```diff
@@ -191,6 +191,36 @@ lemma epi_iff_cokernel_π_eq_zero : epi f ↔ cokernel.π f = 0 :=
 
 end
 
+section opposite
+
+instance exact.op [exact f g] : exact g.op f.op :=
+begin
+  rw exact_iff,
+  refine ⟨by simp [← op_comp], _⟩,
+  apply_fun quiver.hom.unop using quiver.hom.unop_inj,
+  simp only [unop_comp, cokernel.π_op, eq_to_hom_refl, kernel.ι_op, category.id_comp,
+    category.assoc, kernel_comp_cokernel_assoc, zero_comp, comp_zero, unop_zero],
+end
+
+/--
+If `exact g.op f.op`, then `exact f g` as well. This is a lemma because changing lemma to instance
+causes "maximum class-instance resolution depth has been reached" in `pseudoelements.lean`.
+-/
+lemma exact.unop [e : exact g.op f.op] : exact f g :=
```

Comment:
I don't think this is the lemma that @adamtopaz had in mind, even though it's a useful result.

1. I guess Adam wanted `{X Y Z : C^op} (f : X \hom Y) (g : Y \hom Z) [exact f g] : exact g.unop f.unop`
2. Please rephrase the current lemma as an `iff`, and call it `exact.op_iff`.

Reply:
done